docs/api.md

```diff
@@ -20,9 +20,6 @@ Following is the supported API format for prometheus encode:
          prefix: prefix added to each metric name
          expiryTime: seconds of no-flow to wait before deleting prometheus data item
          tls: TLS configuration for the prometheus endpoint
-             enable: set to true to enable tls for the prometheus endpoint
```

Comment:
why this is removed, should not be connected to this PR???

Reply:
That field was changed from struct to struct ptr and the api-to-doc script doesn't handle such types at the moment.
I've submitted a different PR to solve this: https://github.com/netobserv/flowlogs-pipeline/pull/288